Initial Commit

cjacksondeveloper opened the pull request.

cjacksondeveloper committed:
Initial Commit

cjacksondeveloper committed:
MVPs

cjacksondeveloper committed:
PreStandup

cjacksondeveloper committed:
Done 1st day

cjacksondeveloper committed:
Completed Functional

cjacksondeveloper committed:
Some Styling

VivaCode reviewed (commented):
## React Todo

##MVP
- [x] Built out App, TodoList, Todo and Form so the list displays
- [x] Build out functionality to add an item
- [x] Build toggle completed
- [x] Build functional button to clear completed

## Stretch
- [x] Persist data in local storage
- [ ] Build out a search function
- [ ] Host 

## The Great!
Your code is really good! Clean and concise. You committed frequently. Your styling is simple but pleasant. All functionality is there. 

## Suggested Improvements
- Your commits need to be descriptive of what was added, changed or fixed.

## Rating 2

cjacksondeveloper committed:
Added localstorage and functionality

added the ability to retain todos on window refresh with localStorage. Added a checkbox that collapses todos. Updated some Styling.

cjacksondeveloper committed:
Fixed localStorage

properly implemented usage of window.localStorage with the JSON objecct
